### PR TITLE
Add a function to get nCells/nNodes

### PR DESCRIPTION
**Pull request by akleb**
## Purpose
Adds a function that returns the total number of cells and nodes in a grid.

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Testing
I am not sure what our unit testing procedure is, but I can write one if necessary.

## Checklist

- [ ] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation

### CODE REVIEW

**Review by marcomangano — COMMENTED** [review]
Nice addition, looks good. Just two things:
1. Please run black/flake8 to ensure there are no linting errors and all tests pass
2. Can you add a one-line example of how to call this function from a script? You can add it to the method docstring.
Thanks!

**Review by nwu63 — COMMENTED** [review]
Does it make sense to also make the wall boundary cells/nodes part a separate function for consistency? There are no tests right now, but if you are familiar with `unittest` it'd be good to start some.

File: cgnsutilities/cgnsutilities.py [hunk 1]
```diff
@@ -51,15 +51,22 @@ def __init__(self):
         self.name = "domain"
         self.cellDim = 3
 
-    def printInfo(self):
-        """Print some information on the mesh to screen. Specifically
-        information needed by the drag prediction workshop"""
+    def getCellsNodes(self):
```
Comment (eirikurj) [inline cgnsutilities/cgnsutilities.py:54]: Maybe use a more descriptive function name like `getTotalCellsAndNodes`?

File: cgnsutilities/cgnsutilities.py [hunk 2]
```diff
@@ -51,15 +51,22 @@ def __init__(self):
...
         for blk in self.blocks:
             totalCells += (blk.dims[0] - 1) * (blk.dims[1] - 1) * (blk.dims[2] - 1)
             totalNodes += blk.dims[0] * blk.dims[1] * blk.dims[2]
 
+        return totalCells, totalNodes
+        
+
+    def printInfo(self):
+        """Print some information on the mesh to screen. Specifically
+        information needed by the drag prediction workshop"""
```
Comment (eirikurj) [inline cgnsutilities/cgnsutilities.py:67]: Since this is being updated, please update the docstring with more relevant text.

**Review by eirikurj — CHANGES_REQUESTED** [review]
Looks good, just a couple of minor suggestions. 
As @nwu63 mentioned, it would also be great if we can get some unittests started.

**Comment by nwu63** [comment]
@akleb I reverted your commits and manually reapplied it, since `black` changed a bunch of lines unnecessarily. Please check and make sure I didn't miss anything.

**Comment by akleb** [comment]
@nwu63 yes, that looks like everything

**Review by marcomangano — APPROVED** [review]

**Review by eirikurj — APPROVED** [review]

**Pull request merged**